services/web/src/components/RequestBlock/templates/curl.js

```diff
@@ -0,0 +1,23 @@
+export default function templateCurl({ method, url, file, headers, body }) {
+  const code = [];
```

Comment:
could a multiline template string make this simpler?

Reply:
might be more compact ... but doubt it fill become more maintainable

Reply:
works for me then!